ui/raidboss/data/06-ew/dungeon/ktisis_hyperboreia.txt

```diff
@@ -0,0 +1,172 @@
+209.9 "Heavy Smash"
+222.2 "Skull Dasher"
+232.8 "Frostbite and Seek"
+234.8 "--untargetable--"
+248.9 "Punishing Slice #1"
+249.8 "Punishing Slice #2"
+
+#~~~~~~~~~~~~#
+# LADON LORD #
+#~~~~~~~~~~~~#
+
+# -ii 368 653B
+
+# "Pyric Breath" sync / 1[56]:[^:]*:Ladon Lord:(6486|6487|648A|648B):/
+
+1000.0 "--sync--" sync / 00:0839::Concept Review will be sealed off/ window 1000,5
+1010.3 "Scratch" sync / 1[56]:[^:]*:Ladon Lord:648F:/ window 1010.3,10
+1019.3 "Inhale" sync / 1[56]:[^:]*:Ladon Lord:6484:/
+1025.5 "--sync--" sync / 1[56]:[^:]*:Ladon Lord:6485:/
+1034.2 "Pyric Breath" sync / 1[56]:[^:]*:Ladon Lord:6486:/ window 1034.2,10
+1045.3 "Inhale" sync / 1[56]:[^:]*:Ladon Lord:6484:/
+1051.6 "--sync--" sync / 1[56]:[^:]*:Ladon Lord:6485:/
+1060.1 "Pyric Breath #1" sync / 1[56]:[^:]*:Ladon Lord:6487:/
+1062.2 "Pyric Breath #2" sync / 1[56]:[^:]*:Ladon Lord:6489:/
+1067.3 "--sync--" sync / 1[56]:[^:]*:Ladon Lord:648C:/
+
+# Rotation block.
+
+# TODO: Need extended rotations on this. The Pyric Sphere at 120 may not always be there?
+
+1074.4 "Inhale" sync / 1[56]:[^:]*:Ladon Lord:6484:/ window 15,15
+1077.4 "Pyric Sphere" sync / 1[56]:[^:]*:Pyric Sphere:6490:/
+1079.8 "--sync--" sync / 1[56]:[^:]*:Ladon Lord:6485:/
+1082.5 "Pyric Sphere" sync / 1[56]:[^:]*:Pyric Sphere:6491:/
+1088.9 "Pyric Breath #1" #sync / 1[56]:[^:]*:Ladon Lord:(6486|6487|648A|648B):/
+1091.0 "Pyric Breath #2" #sync / 1[56]:[^:]*:Ladon Lord:(6486|6487|648A|648B):/
+1099.1 "Intimidation" sync / 1[56]:[^:]*:Ladon Lord:648D:/
+1105.3 "--sync--" sync / 1[56]:[^:]*:Ladon Lord:648C:/
+1110.4 "Inhale" sync / 1[56]:[^:]*:Ladon Lord:6484:/ window 30,30
+1115.4 "Pyric Sphere" sync / 1[56]:[^:]*:Pyric Sphere:6490:/
+1120.5 "Pyric Sphere?" sync / 1[56]:[^:]*:Pyric Sphere:6491:/ # Doesn't always happen?
+1122.9 "Pyric Blast" sync / 1[56]:[^:]*:Ladon Lord:648E:/
+1126.2 "--sync--" sync / 1[56]:[^:]*:Ladon Lord:6485:/
+1135.0 "Pyric Breath #1" #sync / 1[56]:[^:]*:Ladon Lord:(6486|6487|648A|648B):/
+1137.1 "Pyric Breath #2" #sync / 1[56]:[^:]*:Ladon Lord:(6486|6487|648A|648B):/
+1146.3 "Scratch" sync / 1[56]:[^:]*:Ladon Lord:648F:/
+1151.4 "--sync--" sync / 1[56]:[^:]*:Ladon Lord:648C:/ window 30,30
+
+1158.8 "Inhale" sync / 1[56]:[^:]*:Ladon Lord:6484:/ window 15,15
+1161.8 "Pyric Sphere" sync / 1[56]:[^:]*:Pyric Sphere:6490:/
+1164.2 "--sync--" sync / 1[56]:[^:]*:Ladon Lord:6485:/
+1166.9 "Pyric Sphere" sync / 1[56]:[^:]*:Pyric Sphere:6491:/
+1173.3 "Pyric Breath #1" #sync / 1[56]:[^:]*:Ladon Lord:(6486|6487|648A|648B):/
+1175.4 "Pyric Breath #2" #sync / 1[56]:[^:]*:Ladon Lord:(6486|6487|648A|648B):/
+1183.5 "Intimidation" sync / 1[56]:[^:]*:Ladon Lord:648D:/
+1189.7 "--sync--" sync / 1[56]:[^:]*:Ladon Lord:648C:/
+1194.8 "Inhale" sync / 1[56]:[^:]*:Ladon Lord:6484:/ window 30,30
+1199.8 "Pyric Sphere" sync / 1[56]:[^:]*:Pyric Sphere:6490:/
+1204.9 "Pyric Sphere?" sync / 1[56]:[^:]*:Pyric Sphere:6491:/ # Doesn't always happen?
```

Comment:
I'm not sure what's going on here.  What does make_timeline or looking at the abilities themselves in the log itself say? Mostly asking if "test" is incorrect here or maybe there's some other ability id here or something else.

Reply:
Sorry, what I meant by this was that one generated timeline didn't have this one appear at 120, while another did. Generating a fresh timeline from each log, I think that may have been an anomaly, since both yours and my logs include it. Maybe the best way to handle this is to just put a slightly larger window on Pyric Blast immediately following?